assets/scss/components/cz-body.scss

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
this is really a general styling not a component thus should go to bootstrap relevant file.